zbsprimarystorage/ansible/zbsp.py

```
@@ -0,0 +1,184 @@
+copy_arg.args = "force=yes"
+zstack_lib_copy = copy(copy_arg, host_post_info)
+if zstack_lib_copy != "changed:False":
+    agent_install_arg = AgentInstallArg(trusted_host, pip_url, virtenv_path, init_install)
+    agent_install_arg.agent_name = "zstacklib"
+    agent_install_arg.agent_root = zbsp_root
+    agent_install_arg.pkg_name = pkg_zstacklib
+    agent_install(agent_install_arg, host_post_info)
+
+
+# name: copy zbs primarystorage agent
+copy_arg = CopyArg()
```

Ruff (0.8.2)

`CopyArg` may be undefined, or defined from star imports

(F405)